server/views/pages/dossierRecallInformation.njk

```diff
@@ -15,13 +15,10 @@
                     }
                 }) }}
             {% endif %}
-            <h1 class="govuk-heading-xl">
+            <h1 class="govuk-heading-l">
                 Create a dossier for {{ person.firstName }} {{ person.lastName }} recall
-
-                <div class="govuk-heading-l govuk-!-margin-top-8">
-                    Recall information
-                </div>
             </h1>
+            <div class="govuk-body-l">Dossier due on {{ recall.dossierTargetDate | dateTime }} </div>
```

Comment:
Can you add to the create-dossier-spec.js integration test, to check for this label?